spark-bigquery-tests/src/main/java/com/google/cloud/spark/bigquery/integration/WriteIntegrationTestBase.java

```diff
@@ -365,7 +392,33 @@ public void testIndirectWriteToBigQueryWithDiffInSchemaNullableFieldAndDisableMo
             "select * from %s.%s",
             testDataset.toString(), TestConstants.DIFF_IN_SCHEMA_DEST_TABLE_NAME);
     int numOfRows = (int) bq.query(QueryJobConfiguration.of(query)).getTotalRows();
-    assertThat(numOfRows).isEqualTo(1);
+    assertThat(numOfRows).isGreaterThan(0);
+  }
+
+  @Test
+  public void testInDirectWriteToBigQueryWithDiffInDescription() throws Exception {
+    assumeThat(writeMethod, equalTo(WriteMethod.INDIRECT));
+    spark.conf().set("temporaryGcsBucket", temporaryGcsBucket);
```

Comment:
don't set globally, put in the write options

Reply:
done